WordPress/src/main/java/org/wordpress/android/viewmodel/activitylog/ActivityLogViewModel.kt

```diff
@@ -288,7 +308,7 @@ class ActivityLogViewModel @Inject constructor(
 
     private fun requestEventsUpdate(isLoadingMore: Boolean) {
         if (canRequestEventsUpdate(isLoadingMore)) {
-            val newStatus = if (isLoadingMore) ActivityLogListStatus.LOADING_MORE else ActivityLogListStatus.FETCHING
+            val newStatus = if (isLoadingMore) LOADING_MORE else ActivityLogListStatus.FETCHING
```

Comment:
Minor (🔍 ): Consider using one alternative or the other, not both.

PS: For enums it is usually suggested to use `ActivityLogListStatus.LOADING_MORE` instead `LOADING_MORE` since this makes sure that imports collisions are avoided and might help the reader understand the import directly. If you agree, I suggest using the explicit `ActivityLogListStatus.XYZ` everywhere, which will also improve the imports. Wdyt?

Reply:
I'm going to leave this as is because it's not part of this PR, but a result of a merge. There may be more coming.